src/assets/crubrand/generic/mixins/_sizing.scss

```diff
@@ -29,7 +31,7 @@ $sizeValue: strip-units(16px);
 
             @if $unit == "px" {  /* 7 */
                 $px : append($px, $value);
-                $rem: append($rem, ($val / $sizeValue + rem));
+                $rem: append($rem, math.div($val, $sizeValue + rem));
```

Comment:
I think you have the order of operations here wrong. https://sass-lang.com/documentation/operators#order-of-operations

Reply:
yeah, I'm realizing now that I kept thinking of the additions as part of the right-side of the `/` operator.  I know order of operations, but I guess I was imagining parentheses around `$sizeValue + rem`.